File: preview/hosted_numbers/hosted_number_order/update-default/update-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

hosted_number_order = @client.preview.hosted_numbers
  .hosted_number_orders('HRaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .update(friendly_name: 'friendly_name')

puts hosted_number_order.sid

File: preview/hostedNumbers/hosted_number_order/update-default/update-default.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$hosted_number_order = $twilio->preview->hostedNumbers->hostedNumberOrders("HRaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
                                                      ->update(array(
                                                                   'friendlyName' => "friendlyName"
                                                               )
                                                      );

print($hosted_number_order.sid);